examples/2d/invert_scattering.py

```diff
@@ -0,0 +1,80 @@
+"""
+Inverting scattering via mse
+===========================
+This script aims to quantify the information loss for natural images by
+performing a reconstruction of an image from its scattering coefficients
+via a L2-norm minimization.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import torch.nn.functional as F
+from PIL import Image
+from torch import optim
+
+from kymatio import Scattering2D
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+# Load src image
+img_name = "./images/lena_64.png"
+src_img = Image.open(img_name).convert("RGB")
+src_img = np.array(src_img).astype(np.float32)
+src_img = src_img / 255.0
+y_true = np.array(src_img)  # copy of src_img
```

Comment:
`src_img.copy()` but unclear to me why you do this? (You already created two copies by the previous two operations - in case you were afraid of anything modifying the png file :))